doc/nrf/config_and_build/bootloaders/bootloader_signature_keys.rst

```diff
@@ -1,23 +1,24 @@
-.. _ug_fw_update:
+.. _ug_fw_update_keys:
+.. _bootloader_signature_keys_guide:
 
-Firmware updates
-################
+Signature keys
+##############
 
 .. contents::
    :local:
    :depth: 2
 
-This user guide provides an overview of firmware updates with |NCS|.
+When using :ref:`bootloader` or :doc:`mcuboot:imgtool`, a signature key is used for validating the next image in the boot chain.
```

Comment:
```suggestion
A signature key can be used by :ref:`bootloader` to validate next image in the boot chain.
Dedicated host tools like :doc:`mcuboot:imgtool` can be used to sign application update images.
```

Reply:
Signing firmware is optional in general, imgtool is used in most cases to sign the firmware update

Reply:
I do agree, also there is nuances here. One is where it's validated only once while others not validated and validated on every boot. However I'm uncertain if we should purpose uncertainties into this or document our defaults.

Reply:
I think we may avoid going into too much details, but it would be great to give an overview here

Reply:
> A signature key can be used by :ref:`bootloader` to validate next image in the boot chain.
> Dedicated host tools like :doc:`mcuboot:imgtool` can be used to sign application update images.

How about:
```suggestion
A signature key can be used by :ref:`bootloader` to validate next image in the boot chain.
Dedicated host tools like :doc:`mcuboot:imgtool` can be used to sign application update images, but they can handle the validation differently.
For example, Imgtool signature is only validated once, while sigantures from other tools can be validated on every boot.
```